### PR TITLE
Add gradiant of blue and row record

### PR DESCRIPTION
**Pull request by Suga7772**
Created a beautiful gradient background in the website and a table record. Representing numl google developer student club. Participating for hacktoberfest kindly give constructive review incase of any issue I will be more then glad to sort it out .If all is well, kindly approve and provide my pr with the hacktoberfest-accepted label.

Kudos to open source contribution!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,7 +64,14 @@
...
                     </tr>
-
+                        
+                     <tr>
+                        <td>Hacktoberfest 2022 Letsgoo!</td>
+                        <td>October</td>
+                        <td>October :(</td>
+                        <td>Hackathon</td>
+                        <td><a href="hacktoberfest.com">More about Hacktoberfest!</a></td>
+                     </tr>
```
Comment (souvikrajsingh) [inline index.html]: good work with the gradient @Suga7772 but hacktoberfest has already been added to the list please add something else.

I would suggues you can add Student Programs like Github Campus Expert about internship oppurtunities like Microsoft Engage also Google apprenticeship....

there are so many to add :) kindly make a new PR with the neccessary changes

**Review by souvikrajsingh — COMMENTED** [review]
@Suga7772 Thanks for contributing, i have added the comment to the portion i think that need to be changed.

**Comment by Suga7772** [comment]
i made teh suitable changes ,kindly review and accept if it is valid

**Comment by Suga7772** [comment]
I have closed the conflicting pull request and created a new one with the
changes made accordingly. Kindly review and approve with hacktoberfest
accepted label if deemed fit

**Comment by souvikrajsingh** [comment]
kindly create a new branch when contributing @Suga7772  

Refer to the README.md 

here you're contributing to the master branch

**Pull request closed without merging**